leaflet-providers.js

```diff
@@ -200,14 +200,47 @@
 			options: {
 				maxZoom: 19,
 				variant: 'roads',
-				attribution: 'Imagery from <a href="http://giscience.uni-hd.de/">GIScience Research Group @ University of Heidelberg</a> &mdash; Map data {attribution.OpenStreetMap}'
+				attribution: 'Imagery from <a href="http://giscience.uni-hd.de/">GIScience Research Group @ University of Heidelberg</a> | Map data '
 			},
 			variants: {
-				Roads: 'roads',
+				Roads: {
+				  options: {
+						variant: 'roads',
+						attribution: '{attribution.OpenMapSurfer}{attribution.OpenStreetMap}'
```

Comment:
mixing tabs/spaces here.

Reply:
Good catch, fixed in https://github.com/leaflet-extras/leaflet-providers/pull/322/commits/36649b1c4b0222edb9b332cc9566efc295f2ec64.